### PR TITLE
[DAT-507] - Integrate with the update endpoint

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Integrate with the update ContactInformation endpoint

**Task:** [DAT-507](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-507)

### CODE REVIEW

**Review by bamaro-dev — COMMENTED** [review]
💪

**Comment by jcamposmk** [comment]
:+1:

**Comment by cbernat** [comment]
👍

File: src/services/doppler-user-api-client.ts [hunk 1]
```diff
@@ -101,9 +102,24 @@ export class HttpDopplerUserApiClient implements DopplerUserApiClient {
...
-  public async createOrUpdateContactInformation(
-    values: any,
-  ): Promise<EmptyResultWithoutExpectedErrors> {
-    return { success: true };
+  public async updateContactInformation(values: any): Promise<EmptyResultWithoutExpectedErrors> {
+    try {
+      const { email, jwtToken } = this.getDopplerUserApiConnectionData();
+
+      const response = await this.axios.request({
+        method: 'POST',
```
Comment (cbernat) [inline src/services/doppler-user-api-client.ts:110]: Why not `PUT`?

**Comment by cbernat** [comment]
👍

File: src/services/doppler-user-api-client.ts (hunk 1 above)
Comment (jhoffman-ms) [reply src/services/doppler-user-api-client.ts:110]: when we discussed with Andres about Users Api, we decided that use POST for this endpoint.

File: src/components/UpgradePlanForm/UpgradePlanForm.js [hunk 2]
```diff
@@ -171,7 +171,7 @@ const UpgradePlanForm = ({
                   <FormattedMessage id="common.cancel" />
                 </button>
                 <SubmitButton>
-                  <FormattedMessage id="common.send" />
+                  <FormattedMessage id="upgradePlanForm.update" />
```
Comment (nreal) [inline src/components/UpgradePlanForm/UpgradePlanForm.js:174]: I think that this change should not be in this pull request

**Comment by cbernat** [comment]
Here you have commits that are not yours, could you check please @jhoffman-ms ?

**Comment by cbernat** [comment]
Nice! 🚀

**Pull request merged**